mev-boost/server/service.go

```diff
@@ -348,90 +347,81 @@ func (m *BoostService) verifyConstraintProofs(responsePayload *BidWithInclusionP
 		return errMissingConstraint
 	}
 
-	if len(responsePayload.Proofs) != len(inclusionConstraints) {
+	if responsePayload.Proofs == nil {
+		log.Warn("[BOLT]: Nil proof")
```

Comment:
Probably this log is not needed since an error/warning will be print by the caller if this function returns an error